hypothesis-python/tests/cover/test_conjecture_engine.py

```diff
@@ -188,6 +188,7 @@ def tf(data):
             data.mark_interesting()
 
     runner = ConjectureRunner(tf)
+    runner.stop_time = float("-inf")  # Stop at first bug
```

Comment:
More a thought than a comment for this PR: I've been thinking that we should have some sort of custom "Dev settings" subobject of settings that lets us control low level fiddly details, with big warning labels on it saying "only use this if you know what you're doing, not subject to backwards compatibility guarantees" (main use cases: prototypey features, running experiments using Hypothesis, helping us debug user problems). Controlling multibug discovery would be a sensible toggle to have on there say.

Reply:
People would use it anyway :sob: 

If this would be really useful to maintainers (IMO nobody else should even know about it), I'd just use a global in the relevant internal module.  Add a comment explaining what happens if it's patched to a different value, and we're done!